src/brogue/Rogue.h

```diff
@@ -147,6 +147,11 @@ typedef struct pos {
     short y;
 } pos;
 
+typedef struct box {
+    short width;
+    short height;
+} box;
```

Comment:
This results in an error when compiling for the terminal:
```
In file included from src/platform/term.c:6:0:
src/brogue/Rogue.h:153:3: error: ‘box’ redeclared as different kind of symbol
 } box;
   ^~~
In file included from src/platform/term.c:1:0:
/usr/include/curses.h:613:28: note: previous declaration of ‘box’ was here
 extern NCURSES_EXPORT(int) box (WINDOW *, chtype, chtype);  /* generated */
```
Maybe rename `box` to something else?